### PR TITLE
Add AppVeyor settings for CI on Windows environment

### PR DESCRIPTION
**Pull request by sakama**
This plugin is sometimes used at Windows env. So CI on Windows is necessary.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- C:\Users\appveyor\.gradle
+
+environment:
+  matrix:
+  - JAVA_HOME: C:\Program Files\Java\jdk1.8.0
+  AZURE_ACCOUNT_NAME: sakamatest
+  AZURE_CONTAINER: blobcontainers
+  AZURE_CONTAINER_DIRECTORY: unittests_appveyor
+  AZURE_ACCOUNT_KEY:
+    secure: jrgzEWXOp3+c6r1clT8fRH9ck/aIdnBdVpqQWAwEH/t6w6mGzvBxIt2p0O7dMOowntuIEQBZZ3sIZXT1WzcgGLr11lShl73Y5c93NZJ9oZ7NK1vT4FXaWkRdT4jnAeIn
```
Comment (dmikurube) [inline appveyor.yml:22]: Is it okay to include this credential (?) in the open-sourced code?

Comment (sakama) [reply appveyor.yml:22]: Yes, this one is encrypted values set at [Encrypt configuration data page](https://ci.appveyor.com/tools/encrypt) at AppVeyor console and different from original one.
encryption_data

Comment (dmikurube) [reply appveyor.yml:22]: Got it. lgtm! (Maybe nice to have a comment to describe that, but I leave it to you.)

**Review by dmikurube — APPROVED** [review]

File: appveyor.yml (hunk 1 above)
Comment (sakama) [reply appveyor.yml:22]: Added comment and merged 6a5a62c

**Pull request merged**